src/DynamoCoreWpf/Utilities/ResourceUtilities.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Interop;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+
+namespace Dynamo.Utilities
+{
+    public static class ResourceUtilities
+    {
+
+        public static ImageSource SetImageSource(System.Drawing.Bitmap bmp)
```

Comment:
Like I mentioned in another PR, this should really be called `ConvertToImageSource`.